doc/python.md

```diff
@@ -163,32 +163,42 @@ In the most basic setup, a job ready for packaging would be structured as:
 
 Then, running `python setup.py bdist_egg` will create a file `dist/my_job_package-0.0.0-py2.7.egg`.
 
+## Uploading a Package
+
+Whereas Java and Scala jobs are packaged as Jar files, Python jobs need to be packaged as `Egg` or `Wheel` files. A set
+of example jobs can be build using the `job-server-python/` sbt task `job-server-python/buildPyExamples`. This builds an
+examples Egg and Wheel in `job-server-python/target/python` so we could push this to the server as a job binary:
+
+    curl --data-binary @dist/my_job_package-0.0.0-py2.7.egg \
```

Comment:
As wheel is the recent format and we want to encourage people to use wheels, it might be better to have example commands using wheels instead of eggs. What is your opinion?

Reply:
I have updated all examples with whl files.